react/week4/githubsearch/src/components/UserDetails.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+
+function UserDetails({ user }) {
+  const [details, setDetails] = useState("");
+  const [loading, setLoading] = useState(true);
```

Comment:
I would initialize this as `false` and change it to `true` right before we do the `HTTP` request in our `fetchDetailsData()`
```suggestion
  const [loading, setLoading] = useState(false);
```